.github/workflows/dep_trig_release.yml

```diff
@@ -0,0 +1,64 @@
+# dep_trig_release.yml file will merge it when 
+# the dependabot updates envd and creates a PR,
+# then push a tag to trigger a release
+
+# see also [dependabot.yml]
+
+name: dependabot trigger release
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  dependabot-auto-merge:
+    name: Merge a dependabot PR
+    runs-on: ubuntu-latest
+    # Checking the actor will prevent your Action run failing on non-Dependabot
+    # PRs but also ensures that it only does work for Dependabot PRs.
+    if: github.actor == 'dependabot[bot]'
+    steps:
+    # The step will fail if there's no metadata and so the approval will not occur.
+    - name: Dependabot metadata
+      id: dependabot-metadata
+      uses: dependabot/fetch-metadata@v1.3.5
+      with:
+        github-token: "${{ secrets.GITHUB_TOKEN }}"
+    - name: Wait other jobs(CI and lint)
+      uses: kachick/wait-other-jobs@v1
+      timeout-minutes: 10
+      with:
+        github-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
Is this one necessary if it we use `gh pr merge --auto`?

Reply:
I would test it for a while.